test/JsonApiDotNetCoreMongoDbExampleTests/IntegrationTests/ReadWrite/WorkItemGroup.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using JsonApiDotNetCore.MongoDb.Resources;
+using JsonApiDotNetCore.Resources.Annotations;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace JsonApiDotNetCoreMongoDbExampleTests.IntegrationTests.ReadWrite
+{
+    public class WorkItemGroup : MongoDbIdentifiable
```

Comment:
The 'Items' relationship property was removed from this class. Probably because I've asked before to removed unused code. The difference here is that these models were thought out to be a logical/consistent set of various kinds of attributes and relationships going in various directions, to provide the tests everything they could possibly need. So I'd prefer to not cut out parts any parts here.

This does not apply to the models in the example project itself, which has grown over time and has become messy. We intend to clean that up in the future and aim for the same quality level as in the ReadWrite section.

The same applies to `WriteFakers`, which is supposed to provide a building block for the set of ReadWrite models. I wouldn't cut out parts of that either. (I now realize the name WriteFakers should have been ReadWriteFakers instead, but we forgot to rename it later in the project.)